packages/model-training/src/global/trainingJobList/TrainingJobStatusModal.tsx

```diff
@@ -0,0 +1,396 @@
+      return (
+        <Flex
+          direction={{ default: 'column' }}
+          gap={{ default: 'gapMd' }}
+          style={{ height: '100%' }}
+        >
+          <FlexItem flex={{ default: 'flex_1' }} style={{ overflowY: 'scroll', minHeight: 0 }}>
+            <Content style={{ color: DisabledColor.var }}>
+              No progress information available for this job.
+            </Content>
+          </FlexItem>
+        </Flex>
+      );
+    }
+
+    return (
+      <Flex direction={{ default: 'column' }} gap={{ default: 'gapMd' }} style={{ height: '100%' }}>
+        <FlexItem flex={{ default: 'flex_1' }} style={{ overflowY: 'scroll', minHeight: 0 }}>
+          <TreeView data={treeData} hasGuides />
+        </FlexItem>
+      </Flex>
+    );
+  };
+
+  return (
+    <Modal
+      variant={ModalVariant.small}
+      isOpen
+      onClose={onClose}
+      data-testid="training-job-status-modal"
+    >
+      <ModalHeader
+        data-testid="training-job-status-modal-header"
+        title={
+          <Flex gap={{ default: 'gapMd' }} alignItems={{ default: 'alignItemsCenter' }}>
+            <FlexItem>
+              <Title headingLevel="h2" size="lg">
+                Training Job Status
+              </Title>
+            </FlexItem>
+            <FlexItem>
+              <TrainingJobStatus job={job} jobStatus={status} />
+            </FlexItem>
+          </Flex>
+        }
+      />
+      <ModalBody
+        style={{ height: '400px', display: 'flex', flexDirection: 'column', overflow: 'hidden' }}
+      >
+        <Stack hasGutter style={{ height: '100%', display: 'flex', flexDirection: 'column' }}>
+          {renderStatus()}
+          <StackItem>
+            <Tabs
+              activeKey={activeTab}
+              onSelect={(_ev, tabIndex) => setActiveTab(`${tabIndex}`)}
+              aria-label="status details"
+            >
+              <Tab
+                eventKey={PROGRESS_TAB}
+                aria-label={PROGRESS_TAB}
+                title={<TabTitleText>{PROGRESS_TAB}</TabTitleText>}
+                data-testid="expand-progress"
+              />
+              <Tab
+                eventKey={EVENT_LOG_TAB}
+                aria-label={EVENT_LOG_TAB}
+                title={<TabTitleText>{EVENT_LOG_TAB}</TabTitleText>}
+                data-testid="expand-logs"
+              />
+            </Tabs>
+          </StackItem>
+          <StackItem isFilled style={{ minHeight: 0, overflowY: 'auto' }}>
+            {activeTab === PROGRESS_TAB ? renderProgress() : renderLogs()}
+          </StackItem>
+        </Stack>
+      </ModalBody>
+      <ModalFooter>
+        {isFailed ? (
+          <Button
+            variant="primary"
+            onClick={handleRetry}
+            isDisabled={isRetrying}
+            isLoading={isRetrying}
+            data-testid="retry-job-button"
+          >
+            Retry Job
+          </Button>
+        ) : (inProgress || isPaused) && !isComplete ? (
```

Comment:
the pause resume option should not be shown for inadmissible status

<img width="618" height="591" alt="Image" src="https://github.com/user-attachments/assets/0abb665d-6e3b-4544-882b-bbb11d41bbcb" />